src/guidellm/benchmark/schemas/generative/entrypoints.py

```diff
@@ -283,6 +283,14 @@ def get_default(cls: type[BenchmarkGenerativeTextArgs], field: str) -> Any:
     max_global_error_rate: float | None = Field(
         default=None, description="Maximum global error rate (0-1) before stopping"
     )
+    over_saturation: bool | dict[str, Any] | None = Field(
```

Comment:
After [this comment](https://github.com/vllm-project/guidellm/pull/438/files#r2582557671) drop support for `bool` as its handled higher up.


```suggestion
    over_saturation: dict[str, Any] | None = Field(
```